tutorials/lidar/extract-raster-data-R.Rmd

```diff
@@ -429,15 +421,18 @@ might they differ?
 
 </div>
 
-```{r challenge-code-square-shape, include=TRUE, results="hide", echo=FALSE}
+```{r challenge-code-square-shape, include=TRUE, results="hide", echo=FALSE, eval=FALSE, comment=NA}
```

Comment:
Line 336: you changed to ```{r challenge-code-loops, eval=FALSE, comment=NA, echo=FALSE} instead of adding to the end.  Why simply add, instead of replace here?  Also, moving forward, do we want to recommend one vs the other?  
With challenge code - we don't want the code and (usually) the output to show in the .md but I'm worried that if it isn't evaluated we may miss errors in it when we test/knit.

Reply:
Not necessarily recommending a specific change but let's come up with a general rule of thumb and apply to all challenge code chunks here (I'll add to the template as well).